README.md

```diff
@@ -47,6 +47,16 @@ To run features with a specific tag run
 
 To include or exclude tags see [the cucumber documentation](https://github.com/cucumber/cucumber/wiki/Tags#running-a-subset-of-scenarios)
 
+## Tags
```

Comment:
https://github.gds/gds/digitalmarketplace-jenkins/blob/26c0796acf3e3fed9611c718c1e47fe92e36a261/job_definitions/functional_tests.yml

So we currently have a generic skip tag too which is skipping on jenkins runs rather than runs from a local machine.

Paul and I kind of liked it being on jenkins rather than local because it means devs are reminded every time when running on local that we still haven't fixed some tests. This argument doesn't feel as important for environment specific skip tags.

Maybe we should change it to follow your convention and put it in the Makefile? Consistency is probably preferred. If so, we could also maybe add it to your tags readme section.